src/foam/nanos/crunch/lite/CapableAdapterDAO.js

```diff
@@ -154,9 +166,17 @@ foam.CLASS({
         }
 
         if ( found ) return foundReturn;
-        // payloads.push(obj);
         return ifNotFound(payloads);
       }
+    },
+    {
+      name: 'pubPayloadsUpdate_',
+      flags: ['web'],
+      code: function () {
+        // A publish on propertyChange isn't enough here; slots won't update if
+        //   the object is identical.
+        this.capable.capablePayloads = [ ...this.capable.capablePayloads ];
```

Comment:
So object being identical meaning since its an array, wont update if the arrays elements change so you just reeference it to a new array of the same copied elements?

Reply:
Yeah. I tried `this.capable.propertyChange.pub('capablePayloads', this.capable.capablePayloads$)` put that didn't update everything